File: arcgis_map_sdk_android/android/src/main/kotlin/dev/fluttercommunity/arcgis_map_sdk_android/model/LatLng.kt
package dev.fluttercommunity.arcgis_map_sdk_android.model

import com.arcgismaps.geometry.Point
import com.arcgismaps.geometry.SpatialReference

data class LatLng(
    val longitude: Double,
    val latitude: Double,
)

fun LatLng.toAGSPoint() = Point(longitude, latitude, SpatialReference.wgs84())

File: arcgis_map_sdk_ios/ios/arcgis_map_sdk_ios/Sources/arcgis_map_sdk_ios/Models/LatLng.swift
//
//  LatLng.swift
//  arcgis_map_sdk_ios
//
//  Created by Julian Bissekkou on 10.10.22.
//

import ArcGIS
import Foundation


struct LatLng: Codable {
    let latitude: Double
    let longitude: Double
}

extension LatLng {
    func toAGSPoint() -> Point {
        Point(x: longitude, y: latitude, spatialReference: .wgs84)
    }
}
